node/src/index.ts

```diff
@@ -55,17 +55,50 @@ export class Connection {
   }
 
   /**
-     * Open a table in the database.
-     * @param name The name of the table.
-     */
-  async openTable (name: string): Promise<Table> {
+   * Open a table in the database.
+   *
+   * @param name The name of the table.
+   */
+  async openTable (name: string): Promise<Table>
+  /**
+   * Open a table in the database.
+   *
+   * @param name The name of the table.
+   * @param embeddings An embedding function to use on this Table
+   */
+  async openTable<T> (name: string, embeddings: EmbeddingFunction<T>): Promise<Table<T>>
```

Comment:
is this a merge leftover? why there are two openTable with embeddings, one optional the other is not.

Also `openTable(name:string)` can be covered by `opentable(name, embeddings?)`?